api/api.go

```diff
@@ -2,26 +2,31 @@ package api
 
 import (
 	"context"
-
-	store "github.com/ONSdigital/dp-search-reindex-api/store"
+	"github.com/ONSdigital/log.go/log"
 	"github.com/gorilla/mux"
 )
 
 //JobStoreAPI provides a struct to wrap the api around
 type JobStoreAPI struct {
-	Router   *mux.Router
-	jobStore store.JobStore
+	Router  *mux.Router
+	mongoDB JobStorer
 }
 
 //Setup function sets up the api and returns an api
-func Setup(ctx context.Context, router *mux.Router, jobStore store.JobStore) *JobStoreAPI {
+func Setup(ctx context.Context, router *mux.Router, mongoDB JobStorer) *JobStoreAPI {
 	api := &JobStoreAPI{
-		Router:   router,
-		jobStore: jobStore,
+		Router:  router,
+		mongoDB: mongoDB,
```

Comment:
As above, I'd remove references to mongo from the `api` package and stick to the more generic `jobStore`

Reply:
makes sense - done